DEPLOYING.md

```diff
@@ -0,0 +1,32 @@
+# How to deploy keen.js
+
+## Preparing the release
+
+* Get a cup of coffee and prepare to sit for while.
+* Update the `CHANGELOG` with the new features in this release.
+Make sure you reference the Pull Request/Issue involved with each feature.
+[example](https://github.com/keen/keen-js/blob/431c24b44047adec449184fba0b6a22ca9fdb129/CHANGELOG.md)
+* Update the `README` to reflect the latest version
+* Add and commit the `CHANGELOG` changes and push to master.
+* Run `gulp build` to build out the latest `dist/` files.
+
+## Cutting the Release
+* `git add -f dist/` - force add the `dist/` folder
+* `git tag v{version}` - Tag the version number
+* `git push --tags`
+
+## Deploying to NPM
+
+TODO
+* `npm publish` ???
```

Comment:
After `git add /dist -f`make sure to commit the added files. I usually do this with a basic message like "v{version} release".

`git push --tags` can also be handled with `git push origin v{version}`. May also be worth mentioning that a tag can also be overwritten by adding the `-f` flag.

Last step is to un-commit the `/dist` folder. Keeping those files out of the master branch might be causing more trouble than it is worth, but has so far helped to keep commits small. My technique for this is:

```ssh
$ git reset --soft HEAD~1
$ git rm --cached ./dist
```